demos/python_demos/human_pose_estimation_demo/models.lst

```diff
@@ -0,0 +1,2 @@
+# This file can be used with the --list option of the model downloader.
+single-human-pose-estimation-0001
```

Comment:
It should be `mobilenet-ssd` instead of `mobile-ssd`. The idea is that this list could be used by `downloader.py`

Reply:
Maybe the demo could run with other detectors.

I tried it with `person-detection-retail-0013` and got
`ValueError: could not broadcast input array from shape (1,3,544,320) into shape (1,3,320,544)`
It looks width and height are mixed in
`_, _, self.input_w, self.input_h = self.model.inputs[self._input_layer_name].shape` because https://github.com/opencv/open_model_zoo/blob/develop/models/public/mobilenet-ssd/mobilenet-ssd.md#converted-model says it is (B, C, H, W)

Reply:
Yes you are right. The label of person is different value for different detection models. I have to write about it somewhere?

Reply:
You handle different labels for person by `--label-person`. There is no need to do anything in addition about it.
The problem is mixed H and W.

Reply:
Can the list be extended with
```
pedestrian-and-vehicle-detector-adas-0001
pedestrian-detection-adas-0002
pedestrian-detection-adas-binary-0001
person-detection-retail-????
person-vehicle-bike-detection-crossroad-????
ssd512
ssd_mobilenet_v?_*
ssdlite_mobilenet_v2
```
?
Please, sort the list.